source/Guides/Real-Time-Operating-System-Setup/Real-Time-Linux/build_rt_kernel_using_docker.md

```diff
@@ -2,7 +2,16 @@
 
 ## Introduction
 
-Ubuntu 20.04 x86_64 docker container is used to cross-compile a new kernel. There is a Dockerfile which can be used for that purpose. If you want to build it using gitpod you need to run https://gitpod.io/#https://github.com/ros-realtime/rt-kernel-docker-builder. It will spawn a docker container automatically for you.
+This document explains how to build a real-time kernel using a docker container provided by the ROS Real-Time Working Group. The docker container comes with cross-compilation tools installed, and a ready-to-build RT kernel. This should be the prefered option for those users who simply want to use to cross-compile a new kernel.
+
+## Limitations
+
+For the moment, the tool supports the following options:
+- XXX kernel version and 
+- crosscompilation for aarch64
+- pre-configured kernel settings
+
+In the future, more additional settings will be supported. 
```

Comment:
Probably we can just remove 'In the future' sentence. We could add a new configuration description as we'll add support for it,

Reply:
Removed